presto-main/src/main/java/com/facebook/presto/testing/TestingSqlTime.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.testing;
+
+import com.facebook.presto.spi.ConnectorSession;
+import com.facebook.presto.spi.type.SqlTime;
+import com.facebook.presto.spi.type.SqlTimestamp;
+import com.facebook.presto.spi.type.TimeZoneKey;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+public class TestingSqlTime
+{
+    private TestingSqlTime()
+    {}
```

Comment:
fmt: can be on prev line (i don't want this change, but someone else may want it later)